test/Confluent.Kafka.IntegrationTests/Tests/Consumer_OffsetsForTimes.cs

```diff
@@ -30,12 +30,12 @@ public static partial class Tests
         ///     Basic OffsetsForTimes test on Consumer.
         /// </summary>
         [Theory, MemberData(nameof(KafkaParameters))]
-        public static async Task Consumer_OffsetsForTimes(string bootstrapServers, string topic, string partitionedTopic)
+        public static void Consumer_OffsetsForTimes(string bootstrapServers, string topic, string partitionedTopic)
```

Comment:
this was failing. I believe it's because having it async meant other tests were interfering with it due to interleaved execution. making it synchronous fixed the failure.